modules/ROOT/pages/release-notes/release-notes53.adoc

```diff
@@ -149,15 +152,15 @@ The {productname} 5.3 release includes accompanying changes affecting the {produ
 
 The Java server-side components (`ephox-spelling.war`, `ephox-hyperlinking.war`, and `ephox-image-proxy.war`) have been updated to *version 2.3.0*.
 
-This version requires Java 8 or higher. For information on the removal of Java 7 support, see: <<removalofjava7support,Removal of Java 7 support>>.
+This version requires Java 8 or higher. For information on the removal of Java 7 support, see: xref:removalofjava7support[Removal of Java 7 support].
 
 For information on:
 
-* The Spell Checker Pro plugin, see: link:{baseurl}/plugins/premium/tinymcespellchecker/[Spell Checker Pro plugin].
-* The Link Checker plugin, see: link:{baseurl}/plugins/premium/linkchecker/[Link Checker plugin].
-* The Image Tools plugin, see: link:{baseurl}/plugins/opensource/imagetools/[Image Tools plugin].
-* The Enhanced Media Embed plugin, see: link:{baseurl}/plugins/premium/mediaembed/[Enhanced Media Embed plugin].
-* Deploying the server-side components, see: link:{baseurl}/enterprise/server/[Server-side component installation].
+* The Spell Checker Pro plugin, see: xref:plugins/premium/tinymcespellchecker.adoc[Spell Checker Pro plugin].
+* The Link Checker plugin, see: xref:plugins/premium/linkchecker.adoc[Link Checker plugin].
+* The Image Tools plugin, see: xref:plugins/opensource/imagetools.adoc[Image Tools plugin].
+* The Enhanced Media Embed plugin, see: xref:plugins/premium/mediaembed.adoc[Enhanced Media Embed plugin].
+* Deploying the server-side components, see: xref:enterprise/server/index.adoc[Server-side component installation].
 
 === Removal of Java 7 support
```

Comment:
```suggestion
[[removalofjava7support]]
=== Removal of Java 7 support
```
This needs this id added otherwise the xref above won't resolve.

Reply:
done